redis/src/aio/multiplexed_connection.rs

```diff
@@ -164,8 +190,19 @@ where
         let result = match result {
             // If this push message isn't a reply, we'll pass it as-is to the push manager and stop iterating
             Ok(Value::Push { kind, data }) if !kind.has_reply() => {
+                #[cfg(feature = "cache-aio")]
+                if kind == PushKind::Invalidate {
+                    if let Some(Value::Array(redis_key)) = data.first() {
```

Comment:
I assume that this logic will be repeated in other connection types. It probably shouldn't exist here.